package.json

```diff
@@ -40,6 +39,7 @@
     "jest-enzyme": "^4.0.1",
     "jsjob": "^0.10.13",
     "mocha": "^4.1.0",
+    "noflo-canvas": "0.4.2",
```

Comment:
I'm a little bit confused on why this is noflo-canvas, and not just canvas. But at least it fixed the build, so we can start with that.

Reply:
> I'm a little bit confused on why this is noflo-canvas, and not just canvas. But at least it fixed the build, so we can start with that.

@bergie Some of the other noflo projects (noflo-browser) were using the noflo-canvas.  I was trying to maintain consistency between them.  I believe the canvas dependency requires platform build tools which are a pain to get to work reliably on all platforms, but I don't know for sure why noflo-browser used this.

Reply:
@WilliamRClark noflo-browser depends on noflo-canvas because the `canvas/` components are part of the build (and used in bunch of the noflo-ui example graphs). Here we should be able to just use `canvas` I believe.